servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -1084,7 +1084,7 @@ void RenderForwardMobile::_render_scene(RenderDataRD *p_render_data, const Color
 			}
 		}
 
-		RD::DrawListID draw_list = RD::get_singleton()->draw_list_begin(framebuffer, load_color ? RD::INITIAL_ACTION_LOAD : RD::INITIAL_ACTION_CLEAR, RD::FINAL_ACTION_STORE, RD::INITIAL_ACTION_CLEAR, RD::FINAL_ACTION_STORE, c, 0.0, 0, Rect2(), breadcrumb);
+		RD::DrawListID draw_list = RD::get_singleton()->draw_list_begin(framebuffer, RD::DRAW_CLEAR_ALL, c, 0.0f, 0, Rect2(), breadcrumb);
```

Comment:
I would expect this to produce an error. You are saying that all the color buffers should be cleared, but then you provide no colors in the color array. If a clear is requested, but no clear color is specified, we should throw an error.